lib/orderbook/MatchesProcessor.ts

```diff
@@ -39,33 +39,35 @@ class MatchesProcessor {
       let makerCurrency: string;
       let takerAmount: number;
       let makerAmount: number;
+       // TODO: use configurable amount of subunits/satoshis per token for each currency
       if (taker.quantity > 0) {
         // we are buying the base currency
         takerCurrency = baseCurrency;
         makerCurrency = quoteCurrency;
-        takerAmount = taker.quantity;
-        makerAmount = taker.quantity * maker.price;
+        takerAmount = taker.quantity * 100000000;
```

Comment:
Not sure why multiple by 100000000 here and not working with these amounts in the command line itself

Reply:
The command line & rpc for placing orders uses whole tokens, whereas LND is expecting amounts in satoshis. This is just the number to convert bitcoin to satoshis.